src/language-service/src/schemas/types.ts

```diff
@@ -182,7 +182,7 @@ export type Template = string;
 /**
  * @TJS-pattern ^(?:[01]\d|2[0123]):(?:[012345]\d):(?:[012345]\d)$
  */
-export type Time = string;
+export type Time = string | string[];
```

Comment:
We need to specify the regex for the items in that case:

```
* @items.pattern ^(?:[01]\d|2[0123]):(?:[012345]\d):(?:[012345]\d)$
```

However, this is too generic. Not every part of the whole HA schema accepts multiple items in `Time` by default. This applies to the automation trigger only.

Reply:
I've created #552 to replace this part, as the upcoming Home Assistant 0.115 will have even more changes to this.

So could you revert the pattern/schema changes? So we can merge all other stuff in this PR? 👍

Reply:
SGTM thanks @frenck